Add Security log analyzer page (#555)
neo4j/docs-aura: modules/ROOT/pages/logging/security-log-analyzer.adoc

You can access security logs from *Logs* in the left-hand navigation and selecting *Security* in the tabs at the top.

To switch between instances, use the dropdown menu on the top left.

Security log analyzer is split up in two parts:

* *Summary table* - An aggregated view of security logs, giving a high level overview over the selected time period.
* *Details table* - A detailed view showing individual security events in the selected time period.

To fetch logs, press the *Fetch logs* button and select a time range for the request.
You may optionally select any filters or search text if required, then press *Fetch*.

A summary of security events is returned, showing aggregations per event.
To see the individual security events, click the right arrow at the end of the line to show details for that event.
The details pane shows individual events.

[[fetch-logs]]
== Fetch logs

The *Fetch logs* button opens up a dialog where you select the time range for the security logs. You can choose from the following intervals:

* Last 30 minutes
* Last hour
* Last 2 hours
* Last 6 hours
* Last 24 hours
* Custom time range (start date and end date)

> To fetch the logs after selection of filters and search is done, click the *Fetch* button.

Reviewer: ```suggestion

```
This should really be intuitive enough for the user to figure out.

[NOTE]
====
Security logs are available for a period of {log-retention-days} days, and each request can be for up to {max-request-hours} hours of data.
====

[[filters]]
=== Filters

Use the filter button to load the available filters over the selected time period.
Filters are available for the following fields:

* Status
* Executing User
* Authenticated User
* Driver

[[search]]
=== Search

> Use the search field to search for specific security events or messages.

Reviewer: ```suggestion
You can search for specific security events or messages using the search fiels.
```

| Executing User
| executingUser
| The user who executed the security event.

| Authenticated User
| authenticatedUser
| The user whose credentials were used to authenticate.

| Actions
| -
| Contains an icon button (Arrow Right Circle) to view detailed executions of this specific security message in the Details table. Use this button to filter the Details table to show only executions of the selected security message.
|===

=== Details

.Details table columns
[cols="25,25m,50v"]
|===
| Display Name | Field Name | Description

| Status
| severity
| The status of the security event.

| Message
| message
| The log message.

| Time
| time
| The timestamp when the log was created, including milliseconds.

| Executing User
| executingUser
| The user who executed the security event.

| Authenticated User
| authenticatedUser
| The user whose credentials were used to authenticate.

| Database
| database
| The database for the security event (if applicable).

| Driver
| driver
| The database driver used during the security event.

|===

== Table interactions

=== Sort table

By default, the table is sorted on *Count* for *Summary* and *Status* for *Details*.

> To sort by a column (such as Executing User) click on the column heading.

Reviewer: ```suggestion
Use the column heading to sort by a column (such as Executing User).
```

:log-retention-days: 30
:max-request-hours: 24

[[aura-monitoring]]
= Security log analyzer

label:AuraDB-Professional[]
label:AuraDB-Virtual-Dedicated-Cloud[]

Security log analyzer is a feature that provides a UI to review the security events on an Aura instance.

You can access security logs from *Logs* in the left-hand navigation and selecting *Security* in the tabs at the top.

> To toggle between instances, use the dropdown menu on the top left.

Reviewer 2: "Toggle" to me sounds like picking between two options. Would prefer "select" or "switch".

Security log analyzer is split up in two parts:

* *Summary table* - An aggregated view of security logs, giving a high level overview over the selected time period.
* *Details table* - A detailed view showing individual security events in the selected time period.

To fetch logs, press the *Fetch logs* button. 

> You may optionally select any filters or search text if required, then press *Fetch*.

Reviewer 2: Maybe also mention selecting time window here. Either that, or leave both filters/time window to the section below.

A summary of security events is returned, showing aggregations per event.
To see the individual security events, click the right arrow at the end of the line to show details for that event.
The details pane shows individual events.

[[fetch-logs]]
== Fetch logs

The *Fetch logs* button opens up a dialog where you select the time range for the security logs. You can choose from the following intervals:

* Last 30 minutes
* Last hour
* Last 2 hours
* Last 6 hours
* Last 24 hours
* Custom time range (start date and end date)

> You can also add filters and search before fetching the logs.
> To fetch the logs after selection of filters and search is done, click the *Fetch* button.

Reviewer 2: ```suggestion
```

Reviewer 2: No need to mention filters again IMO
